### PR TITLE
Add support for eQ surveys

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Displays relevant statistics for eQ surveys.
Have also changed the way we display the numbers, the HTML elements are now dynamically created. This is so that we don't create all elements for the numbers regardless and display only those which are applicable.

### How to review 
- Ensure statistics for eQ surveys are displayed.
- Ensure the rest of the app works as expected.

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Build a new front end for eQ surveys](https://trello.com/c/0AyiJOQY/31-build-a-new-front-end-for-eq-surveys)

**Review by ryangrundy7 — DISMISSED** [review]

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
The mock services script is broken.

**Comment by MebinAbraham** [comment]
@AdamHawtin Need more context, this doesn't help me -_-. Also seems to be working fine for me.

File: scripts/mock_services.py [hunk 1]
```diff
@@ -218,6 +245,37 @@ def get_collection_exercises():
...
+                            "actionPlanId": "e71002ac-3575-47eb-b87f-cd9db92bf9a7",
+                            "sampleUnitType": "B"
+                        },
+                        {
+                            "actionPlanId": "0009e978-0932-463b-a2a1-b45cb3ffcb2a",
+                            "sampleUnitType": "BI"
+                        }
+                    ],
+                    "exerciseRef": "201812",
+                    "userDescription": "Quarterly Business Survey",
```
Comment (AdamHawtin) [inline scripts/mock_services.py]: This should be a description of the dates/period of the CE, not the survey.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: scripts/mock_services.py (hunk 1 above)
Comment (MebinAbraham) [reply scripts/mock_services.py]: Oh ya

File: app/common/survey_metadata.py [hunk 2]
```diff
@@ -8,16 +8,37 @@
 
 
 def map_surveys_to_collection_exercises(surveys, collection_exercises) -> list:
+    seft_surveys = {
+        'cb8accda-6118-4d3b-85a3-149e28960c54': 'Brick',
```
Comment (ryangrundy7) [inline app/common/survey_metadata.py]: Brick should be changed to Bricks

**Review by AdamHawtin — APPROVED** [review]

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**